cluster/ci/config/prow/prow_crd.yaml

```diff
@@ -13,3 +13,41 @@ spec:
     listKind: ProwJobList
     shortNames:
     - pj
+  validation:
+    openAPIV3Schema:
+      properties:
+        spec:
+          properties:
+            max_concurrency:
+              type: integer
+              minimum: 0
+            type:
+              type: string
+              enum:
+              - "presubmit"
+              - "postsubmit"
+              - "periodic"
+              - "batch"
+        status:
+          properties:
+            state:
+              type: string
+              enum:
+              - "triggered"
+              - "pending"
+              - "success"
+              - "failure"
+              - "aborted"
+              - "error"
+          anyOf:
+          - not:
```

Comment:
why this anyOf and not? If `state` is given, `completionTime` is required?

Reply:
We want completionTime to be set when state is terminal (success, failure, error, aborted).

Reply:
sounds fine.